scipy/signal/spectral.py

```diff
@@ -26,30 +27,34 @@ def periodogram(x, fs=1.0, window=None, nfft=None, detrend='constant',
     x : array_like
         Time series of measurement values
     fs : float, optional
-        Sampling frequency of the `x` time series. Defaults to 1.0.
+        Sampling frequency of the ``x`` time series. Defaults to 1.0.
```

Comment:
We usually use single quotes for these even though the references don't populate properly